.buildkite/scripts/build_push_docker_image.sh

```diff
@@ -1,16 +1,13 @@
 #!/bin/bash
 
-set -euo pipefail
+set -euox pipefail
```

Comment:
maybe revert?

Reply:
sure, I just was making the final test...
reverted.